tests/playwright/src/ai-lab-extension-installation.spec.ts

```diff
@@ -0,0 +1,88 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { afterAll, beforeAll, beforeEach, describe, test } from 'vitest';
+import type { DashboardPage, SettingsBar, RunnerTestContext } from '@podman-desktop/tests-playwright';
+import {
+  NavigationBar,
+  SettingsExtensionsPage,
+  WelcomePage,
+  PodmanDesktopRunner,
+} from '@podman-desktop/tests-playwright';
+
+const AI_LAB_EXTENSION_OCI_IMAGE: string = 'ghcr.io/projectatomic/ai-studio:nightly';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+
+let navigationBar: NavigationBar;
+let dashboardPage: DashboardPage;
+let settingsBar: SettingsBar;
+let extensionsPage: SettingsExtensionsPage;
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('podman-desktop-ai-lab-extension-installation');
```

Comment:
```suggestion
  pdRunner.setVideoAndTraceName('podman-desktop-extension-ai-lab-installation');
```
(to mimic repository name)